src/css/sidebar.css

```diff
@@ -0,0 +1,57 @@
+.panel-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.sidebar-container {
+    height: 100%;
+    background-color: white;
+    z-index: 2;
+}
+
+.sidebar-content {
+    overflow-y: auto;
+    height: calc( 100% - 52px );
+}
+
+.sidebar-container.closed {
+    display: none;
+}
+
+.toggleButton {
```

Comment:
Style SidebarButton to match [the Figma design](https://www.figma.com/file/vP3gxGjxW34i5gNMpu6sDW/Design-V1?node-id=1472%3A11484).